build.sh

```diff
@@ -14,7 +14,7 @@ if [ "$(uname)" == "Darwin" ]; then
 
 elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
     # Running on Linux.
-    # Check whether podman is available, else faill back to docker
+    # Check whether podman is available, else fail back to docker
```

Comment:
I think that should be 'fall' here.

Reply:
Duh. You're totally right. Fixed and force-pushed a change in my commit.